wiki/Tournaments/TWC/2025/en.md

```diff
@@ -134,19 +134,19 @@ The complete sign-up list can be found [here](https://gist.github.com/LeoFLT/bca
 
 ### Semifinals
 
-**[Download the mappack here (113 MB)](https://leoflt.com/s/twc2025-semifinals-mappack)**\
```

Comment:
can we not use private hosting going forward, ever, please? you can ask if you need something uploaded.

Reply:
cc @ppy/wiki for blocking during review